### ISSUE TITLE
JWT expiry throws 500 error 

### ISSUE THREAD

**Issue by tim-field** [graphile/crystal#1847]
### Summary

I've got PostGraphile handling my JWT's in PostgreSQL per the examples in the documentation. 

When the token expires the server responds with a `500` status code and an `Unknown error occurred` in the returned errors object

```
{
    "errors": [
        {
            "message": "Unknown error occurred",
            "extensions": {
                "stack": [
                    "TokenExpiredError: jwt expired",
                    // error stack here
                ]
            }
        }
    ]
}
```

### Steps to reproduce

Send a request with an expired JWT token ( when postgraphile is handling JWTs ) 

### Expected results

Ideally a `401` status code and Token expiry specific error in the graphql error object.

### Actual results

a `500` status with an `Unknown error occurred` message

```
    "errors": [
        {
            "message": "Unknown error occurred",
            "extensions": {
                "stack": [
                    "TokenExpiredError: jwt expired",
                    // error stack here
                ]
            }
        }
    ]
}
```

### Additional context

    `"postgraphile": "^5.0.0-beta.15"`

@benjie  thinks this is a bug and suggested creating an issue here.

https://discord.com/channels/489127045289476126/1170129161516486756/1170163030731018380

**Comment by benjie** [comment]
Specifically this seems to be an issue with the `PgLazyJWTPreset`

**Comment by benjie** [comment]
This was fixed in #1914

**Comment by benjie** [comment]
Though now it returns 200 which also isn't ideal...
